composer.json

```diff
@@ -20,7 +20,12 @@
     },
     "require": {
         "dachcom-digital/dynamic-search": "^0.6",
-        "pimcore/pimcore": "^5.8.0",
-        "vdb/php-spider": "^0.5"
+        "pimcore/pimcore": "^5.8.0 | ^6.0",
+        "vdb/php-spider": "dev-master"
```

Comment:
Please change `php-spider` to `^0.4` since it was just a typo.

Reply:
done